packages/flutter/test/material/divider_theme_test.dart

```diff
@@ -0,0 +1,234 @@
+      final DividerThemeData dividerTheme = _dividerTheme();
+      await tester.pumpWidget(MaterialApp(
+        theme: ThemeData(dividerTheme: dividerTheme),
+        home: Scaffold(
+          body: const Divider(),
+        ),
+      ));
+
+      final RenderBox box = tester.firstRenderObject(find.byType(Divider));
+      expect(box.size.height, dividerTheme.space);
+
+      final Container container = tester.widget(find.byType(Container));
+      final BoxDecoration decoration = container.decoration;
+      expect(decoration.border.bottom.width, dividerTheme.thickness);
+      expect(decoration.border.bottom.color, dividerTheme.color);
+
+      final Rect dividerRect = tester.getRect(find.byType(Divider));
+      final Rect lineRect = tester.getRect(find.byType(DecoratedBox));
+      expect(lineRect.left, dividerRect.left + dividerTheme.indent);
+      expect(lineRect.right, dividerRect.right - dividerTheme.endIndent);
+    });
+
+    testWidgets('Widget properties take priority over theme', (WidgetTester tester) async {
+      const Color color = Colors.purple;
+      const double height = 10.0;
+      const double thickness = 5.0;
+      const double indent = 8.0;
+      const double endIndent = 9.0;
+
+      final DividerThemeData dividerTheme = _dividerTheme();
+      await tester.pumpWidget(MaterialApp(
+        theme: ThemeData(dividerTheme: dividerTheme),
+        home: Scaffold(
+          body: const Divider(
+            color: color,
+            height: height,
+            thickness: thickness,
+            indent: indent,
+            endIndent: endIndent,
+          ),
+        ),
+      ));
+
+      final RenderBox box = tester.firstRenderObject(find.byType(Divider));
+      expect(box.size.height, height);
+
+      final Container container = tester.widget(find.byType(Container));
+      final BoxDecoration decoration = container.decoration;
+      expect(decoration.border.bottom.width, thickness);
+      expect(decoration.border.bottom.color, color);
+
+      final Rect dividerRect = tester.getRect(find.byType(Divider));
+      final Rect lineRect = tester.getRect(find.byType(DecoratedBox));
+      expect(lineRect.left, dividerRect.left + indent);
+      expect(lineRect.right, dividerRect.right - endIndent);
+    });
+  });
+
+  group('Vertical Divider', () {
+    testWidgets('Passing no DividerThemeData returns defaults', (WidgetTester tester) async {
+      await tester.pumpWidget(MaterialApp(
+        home: Scaffold(
+          body: const VerticalDivider(),
+        ),
+      ));
+
+      final RenderBox box = tester.firstRenderObject(find.byType(VerticalDivider));
+      expect(box.size.width, 16.0);
+
+      final Container container = tester.widget(find.byType(Container));
+      final BoxDecoration decoration = container.decoration;
+      final Border border = decoration.border;
+      expect(border.left.width, 0.0);
+      expect(border.left.color, const Color(0x1F000000));
```

Comment:
Where is this default coming from. Would it be better to test against the value it is stored as?

Reply:
The value comes from the default value that ThemeData creates. Even though I don't specify a theme, a fallback theme is created. 

I've updated the test to check against that value instead.